src/loadData.js

```diff
@@ -0,0 +1,84 @@
+function loadName(card, info, i){
+    let name = document.createElement('h5');
+    name.className = 'card-title'
+    name.innerText = info[i].name;
+    card.appendChild(name);
+}
+
+function loadTitle(card, info, i){
+    let name = document.createElement('h5');
+    name.className = 'card-title'
+    name.innerText = info[i].title;
+    card.appendChild(name);
```

Comment:
looks like duplication of `loadName`. follow DRY please

Reply:
I made refactoring of this and its removed repeated parts.